SFS - UZFS.json

```diff
@@ -275,7 +275,7 @@
           "useBackend": false
         }
       ],
-      "title": "Average Request Size in Bytes",
+      "title": "Max latency in 1 Minute",
```

Comment:
The current time range uses `[$__rate_interval]` for flexibility, allowing Grafana to adapt the time window. For example, when fetching 24-hour monitoring data, the time window will be larger, resulting in fewer points on the monitoring panel and avoiding excessive density.